app/services/hyrax/listeners/active_fedora_acl_index_listener.rb

```diff
@@ -11,7 +11,9 @@ class ActiveFedoraAclIndexListener
       ##
       # Re-index the resource for the updated ACL.
       #
-      # @param event [Dry::Event]
+      # Called when 'object.acl.updated' event is published
+      # @param [Dry::Events::Event] event
+      # @return []
```

Comment:
Yard uses `void` for "undefined" in these cases

```suggestion
      # @return [void]
```